querydsl-core/src/main/java/com/mysema/query/types/ExpressionUtils.java

```diff
@@ -43,6 +43,193 @@ private UnderscoreTemplates() {
 
     private static final Templates TEMPLATES = new UnderscoreTemplates();
 
+
+    /**
+     * Create a new Operation expression
+     *
+     * @param type type of expression
+     * @param operator operator
+     * @param args operation arguments
+     * @return operation expression
+     */
+    public static <T> Operation<T> operation(Class<? extends T> type, Operator operator,
+                                             Expression<?>... args) {
+        return operation(type, operator, ImmutableList.copyOf(args));
+    }
+
+    /**
+     * Create a new Operation expression
+     *
+     * @param type type of expression
+     * @param operator operator
+     * @param args operation arguments
+     * @return operation expression
+     */
+    @SuppressWarnings("unchecked")
+    public static <T> Operation<T> operation(Class<? extends T> type, Operator operator,
+                                             ImmutableList<Expression<?>> args) {
+        if (type.equals(Boolean.class)) {
+            return (Operation<T>) new PredicateOperation(operator, args);
+        } else {
+            return new OperationImpl<T>(type, operator, args);
+        }
+    }
+
+    /**
+     * Create a new Operation expression
+     *
+     * @param operator operator
+     * @param args operation arguments
+     * @return operation expression
+     */
+    public static PredicateOperation predicate(Operator operator, Expression<?>... args) {
+        return predicate(operator, ImmutableList.copyOf(args));
+    }
+
+    /**
+     * Create a new Operation expression
+     *
+     * @param operator operator
+     * @param args operation arguments
+     * @return operation expression
+     */
+    public static PredicateOperation predicate(Operator operator, ImmutableList<Expression<?>> args) {
+        return new PredicateOperation(operator, args);
+    }
+
+    /**
+     * Create a new Path expression
+     *
+     * @param type type of expression
+     * @param variable variable name
+     * @return path expression
+     */
+    public static <T> Path<T> path(Class<? extends T> type, String variable) {
+        return new PathImpl<T>(type, variable);
+    }
+
+    /**
+     * Create a new Path expression
+     *
+     * @param type type of expression
+     * @param parent parent path
+     * @param property property name
+     * @return property path
```

Comment:
Why does this say `property path` and the other places is `path expression`?
I think `path expression` sounds more in line with `ExpressionUtils`.